README.md

```diff
@@ -118,6 +118,49 @@ example :
   - your-page-doc-name
 ```
 
+#### Create a release
+
+> Define new version using [semantic versioning](https://semver.org/)
+
+> Create a new local branch `release/X.Y.Z` from master :
```

Comment:
```suggestion
- Create a new local branch `release/X.Y.Z` from master 
```

`>` means blockquote in markdown. I think you want bullet points instead (`-` or `*`)